boards/nrf51dk/board.c

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2014 Freie Universität Berlin
```

Comment:
feel free to change this to 2016 and put your name/company in as copyright holder...